### PR TITLE
fix: update entry_points to use gradio.cli, not gradio.reload

### PR DESCRIPTION
**Pull request by xhiroga**
Fix https://github.com/conda-forge/gradio-feedstock/issues/48

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,61 +10,67 @@ source:
...
-  license_file: LICENSE
+  license: MIT AND Apache-2.0
+  license_file:
+    - LICENSE
+    - gradio/node/dev/node_modules/esbuild-wasm/LICENSE.md
 
 extra:
   recipe-maintainers:
     - Sclare87
+    - xhiroga
```
Comment (xhiroga) [inline recipe/meta.yaml:76]: Please note that my user ID is listed as a maintainer, but it was auto-generated by grayskull and I have no specific preference regarding this.

**Comment by Sclare87** [comment]
Thank you for the pull request.
Looks fine to me and i want to merge it.

But before, can u please increment the build number (the gradio version hasn't changed so we need to trigger a new build for the current version)

File: recipe/meta.yaml (hunk 1 above)
Comment (Sclare87) [reply recipe/meta.yaml:76]: I am not offical linked with the development of gradio, i only maintain the conda-forge packages (update requirements if needed and so on).
If you dont like to be added, than you can remove it.

**Comment by xhiroga** [comment]
@Sclare87 
Thank you for your prompt response to the pull request. I appreciate your willingness to merge it. Regarding the inclusion of my name, I've decided to keep it as is and will continue to watch the repository with interest.

Also, thank you for taking the time to adjust the build number. I apologize for my lack of understanding regarding this aspect and am grateful for your attention to detail.

I have reviewed everything thoroughly and am happy to confirm that it's all set for merging. Thank you once again for your cooperation and support!

**Pull request merged**